packages/schemas/src/seeds/tenant.ts

```diff
@@ -1,12 +1,22 @@
 import type { InferModelType } from '@withtyped/server';
+import { z } from 'zod';
 
 import type { Tenants } from '../models/tenants.js';
+import { TenantTag } from '../types/tenant.js';
 
 export const defaultTenantId = 'default';
 export const adminTenantId = 'admin';
 
-/**
- * `createModel` from @withtyped/server can not properly infer the model
- * type, manually define it here for now.
- */
-export type TenantModel = Pick<InferModelType<typeof Tenants>, 'id' | 'dbUser' | 'dbUserPassword'>;
+// TODO: fix the type definition once `withtyped/server` can handle `Date` type properly.
+export type TenantModel = InferModelType<typeof Tenants>;
```

Comment:
what's the relation between Date handling and model inference here?

Reply:
This comment is outdated, removed.